cudapoa/src/application_parameters.cpp

```diff
@@ -63,10 +62,10 @@ ApplicationParameters::ApplicationParameters(int argc, char* argv[])
         case 'a':
             msa = true;
             break;
-        case 'f':
-            banded = false;
-            break;
         case 'b':
+            band_mode = std::stoi(optarg);
```

Comment:
can this `band_mode` variable be made a `BandMode` type? then in this conditional we set the `BandMode` to the relevant type based on `stoi`, and all checks downstream can be based on `band_mode == BandMode::adaptive?` etc?

Reply:
done.